_metadata/repos.conf.d/fedora/provider.conf

```diff
@@ -1,40 +1 @@
-linux/core/
-linux/development/
-linux/extras/
-linux/modular/
-linux/releases/test/
-linux/releases/7/
-linux/releases/8/
-linux/releases/9/
-linux/releases/1*
-linux/releases/20/
-linux/releases/21/
-linux/releases/22/
-linux/releases/23/
-linux/releases/24/
-linux/releases/25/
-linux/releases/*/CloudImages/
-linux/releases/*/Docker/
-linux/releases/*/Server/
-linux/releases/*/Spins/
-linux/releases/*/Workstation/
-linux/releases/*/Everything/armhfp/
-linux/releases/*/Everything/i386/
-linux/releases/*/Everything/source/
-linux/releases/*/Everything/x86_64/debug/
-linux/releases/*/Everything/x86_64/iso/
-linux/updates/testing/
-linux/updates/7/
-linux/updates/8/
-linux/updates/9/
-linux/updates/1*
-linux/updates/20
-linux/updates/21
-linux/updates/22
-linux/updates/23
-linux/updates/24
-linux/updates/25
-linux/updates/*/SRPMS/
-linux/updates/*/armhfp/
-linux/updates/*/i386/
-linux/updates/*/x86_64/debug/
+linux/releases/*/CloudImages/
```

Comment:
Should be including cloud images and ISOs, so we can cut down on separate repos. I still don't think we need testing or source RPMS, Also, just like with CentOS: "Everything" is an aggregate of everything in "Workstation" and "Server", there is no need to grab all three. "Spins" is just a directory containing ISO files of several different variants of Fedora ISOs with different Desktop environments. This would be costly to download compared to the benefit it adds. I think this should continue to be excluded as well.

Reply:
i agree with @codrcodz, I dont see any need to grab any of the old fedora core versions. Start with the oldest one requested on the network and go up from there.